cmd/cmd_internal_test.go

```diff
@@ -62,7 +62,7 @@ func TestCmdFlags(t *testing.T) {
 					LokiService: "charon",
 				},
 				P2P: p2p.Config{
-					Relays:   []string{"https://0.relay.obol.tech", "https://1.relay.obol.tech"},
+					Relays:   []string{"https://0.relay.obol.tech", "https://2.relay.obol.dev", "https://1.relay.obol.tech"},
```

Comment:
Is this order intentional? 0, 2, 1, not 0, 1, 2?

Reply:
Both 0.relay and 1.relay use the same infra stack, and the goal is to failover to the obol.dev stack (AWS + Netlify DNS) if obol.tech stack (GCP + Cloudflare DNS) is unavailable.